packages/translate/system-test/translate.js

```diff
@@ -23,16 +23,8 @@ var prop = require('propprop');
 var env = require('../../../system-test/env.js');
 var Translate = require('../');
 
-var API_KEY = process.env.GCLOUD_TESTS_API_KEY;
-
-// Only run the tests if there is an API key to test with.
-(API_KEY ? describe : describe.skip)('translate', function() {
-  if (!API_KEY) {
-    // The test runner still executes this function, even if it is skipped.
-    return;
-  }
-
-  var translate = new Translate(extend({}, env, { key: API_KEY }));
+describe('translate', function() {
```

Comment:
We should make sure to test that authenticating with an API key works. Just a test that uses the key if the env var is set, `this.skip()`s otherwise, and hits an endpoint to check that it doesn't error.